### DISCUSSION TITLE
Add solidity language

### DISCUSSION THREAD

**Discussion by devpolo** [community/community#8576] [Other Feature Feedback, Questions, & Ideas] [answered]
´language:solidity´

**Comment by tclem** [comment]
This does work, but there's not a lot of public solidity code currently in the index. See results at https://cs.github.com/?scopeName=All+repos&scope=&q=language%3Asolidity.

**Comment by devpolo** [comment, marked as answer]
Yes it is! I'm concerne about why I'm not able to see it on autocomplete like other languages?

**Reply by tclem** [reply]
Ah, good feedback. The language suggestions aren't too smart right now and only pull from a small list of popular ones.
